pkg/openapi/openapi_generated.go

```diff
@@ -1577,6 +1569,27 @@ func GetOpenAPIDefinitions(ref openapi.ReferenceCallback) map[string]openapi.Ope
 			},
 			Dependencies: []string{},
 		},
+		"k8s.io/kops/pkg/apis/kops/v1alpha1.Assets": {
```

Comment:
Not entirely sure, but my guess is that you're just picking up that it wasn't run before...  Don't worry about it anyway, although it highlights that we need to get this under CI testing lest people forget.